classes/ImageManager.php

```diff
@@ -306,7 +306,12 @@ public static function resize(
 
         // If the output is PNG, fill with transparency. Else fill with white background.
         if ($destinationFileType == 'png' || $destinationFileType == 'webp' || $destinationFileType == 'avif') {
```

Comment:
```suggestion
        if (in_array($destinationFileType, ['png', 'webp', 'avif']) {
```

Reply:
My PR doesn't care about this line of code, thanks.

Reply:
😢

Reply:
@SharakPL don't cry, I've done it for you

Reply:
@SharakPL you have now to convince @Hlavtox that this pr is relevant.

Reply:
@jf-viguier everybody here knows it's relevant. It's just about doing it right way.

btw please don't name commits after me 🤣